Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/sources/fallback/DefaultAppConfigSourceTest.kt

```diff
@@ -81,6 +82,7 @@ class DefaultAppConfigSourceTest : BaseIOTest() {
 
     @Test
     fun `exceptions when getting the default config are rethrown`() = runBlockingTest {
+        every { assetManager.open("default_app_config_android.bin") } throws FileNotFoundException("default_app_config_android.bin does not exist")
```

Comment:
Why did you add this? The original mock specifies a file to be read, which doesn't exist without `configFile.writeBytes(APPCONFIG_RAW)` and should thus throw an exception here without requiring us to mock an exception being thrown.

Reply:
But it doesn't and even fails on release 1.15.
Just tried to fix this test for test pipe.